cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSICKANSink.java

```diff
@@ -364,13 +404,26 @@ public String getResName(boolean enableLowercase) {
         } // getResName
 
         public void initialize(NGSIEvent event) throws CygnusBadConfiguration {
+        	LOGGER.debug("[" + this.getClass() + "] INITIALIZE");
```

Comment:
use `getName` method instead of `this.getClass`

Reply:
Fixed in f1f9737cf4ef5d7a113be681a47790b8d860c0f3